sample/authentication/crypto.go

```diff
@@ -122,19 +125,38 @@ func (a *PublicAuthenScheme) VerifyAuthenticationTag(m []byte, sig []byte, pubKe
 	return nil
 }
 
-// SGXUSIGAuthenticationScheme impelements AuthenticationScheme
-// interface by utilizing SGX USIG to create/verify authentication
-// tags.
+// usigKeyFingerprint is a SHA256 hash of the USIG public key.
+type usigKeyFingerprint [sha256.Size]byte
```

Comment:
Maybe you want to use a shorter hash output or truncate the output? SGX ECDSA is 256-bit, so the public key is 512-bit long. Your fingerprint only saves half the space.

Reply:
I'm thinking to truncate this to 64-bit fingerprint.